### ISSUE TITLE
pretrained vilbert model not found

### ISSUE THREAD

**Issue by arjunakula** [jiasenlu/vilbert_beta#13]
Hi Jiasen,
    I am trying to train (fine-tune) the downstream task for refcoco+... The code throws an error saying that pretrained vilbert model is not found. Is there a link where I can download the pretrained vilbert model.

 The error that I get is the following:
ERROR - vilbert.vilbert -   Model name 'save/bert_base_6_layer_6_connect_
freeze_0/pytorch_model_8.bin' was not found in model name list...

Thanks a lot!

**Comment by arjunakula** [comment]
Just simply renaming bert_base_6_layer_6_connect_
freeze_0/pytorch_model_8.bin to bert_base_6_layer_6_connect_
freeze_0/pytorch_model_9.bin worked. closing the issue.

**Comment by chchenhui** [comment]
I also encountered this problem. But the error shows the bin document I downloaded was not found in model name. I am not sure which name I should use...
<img width="1181" alt="image" src="https://user-images.githubusercontent.com/13835142/64758769-03e27a80-d568-11e9-9310-2150d4f258bc.png">
As shown in this screenshot, pytorch_model_19.bin is exactly the document I downloaded and the name is the same with the input param.
